snapwell/snapwell_main.py

```diff
@@ -302,18 +333,28 @@ def run(self):
         conftime = round(confstop - fullstart, 2)
         logging.info("\n\nConfiguration completed in %s sec.\n", str(conftime))
 
-        runner.main_loop()
+        success = runner.main_loop()
 
         fullstop = time()
         fullsec = round(fullstop - fullstart, 2)
         logging.info("snapwell completed in %s seconds", str(fullsec))
 
+        if not success:
+            logging.error("Snapwell completed, but errors occurred")
+            return -1
```

Comment:
I _think_ the reason for the previous behavior is that if a single part of the forward fails, the entire forward model will fail, so the thinking has probably been, that if we can limp through this, lets. From an executable point of view, that is definitely wrong, but might be correct(ish) behavior for a forward model.

Reply:
I think its not too bad if we go through every well-file and attempt to apply the transformation to it even though some of them fails, but its important that we exit with a non-zero value in that case.

Reply:
> but its important that we exit with a non-zero value in that case.

I very much agree with that, but think we should also run it by the users.